lua/astrocommunity/completion/codeium-vim/init.lua

```diff
@@ -1,17 +1,45 @@
 return {
   "Exafunction/codeium.vim",
-  event = "User AstroFile",
```

Comment:
In my testing, ive noticed that this event no longer works? Cant explain why, havent done the research on it, kinda tired rn. @mehalter Is it a potential bug from AstroNvim? Could look sometime later if you arent intrested.

Reply:
Lazy loading with vim pugins is wonky. We should just explicitly disable it with
`lazy = false` instead.

Reply:
Would rather go with the `BufEnter` event and it works, thats how they used it in their [examples](https://github.com/Exafunction/codeium.vim/blob/7a01ffb1e6538fe634f294908a50a7b680039284/README.md#-lazy), also ive implemented it already in this pr.

Reply:
Sure if you've tested it lets go for that.